services/users.service.ts

```diff
@@ -378,11 +379,14 @@ export default class UsersService extends moleculer.Service {
     };
 
     if (user?.id) {
-      return ctx.call('users.update', {
-        id: user.id,
-        ...dataToSave,
-        scope,
-      });
+      return ctx.call(
+        'users.update',
+        {
+          id: user.id,
+          ...dataToSave,
+        },
+        { scope } as any
+      );
```

Comment:
kadangi tu `users.service`, tai siulau naudot `this.updateEntity(ctx, params, {scope: ...})` ir panaikinti is database mixino ta `update` dali.